sample/src/main/kotlin/com/slack/circuit/sample/petdetail/PetDetail.kt

```diff
@@ -86,7 +86,9 @@ constructor(
       )
     }
 
-    render(state) { /* nothing to do yet! */}
+//    LaunchedEffect(this) { /* nothing to do yet */ }
```

Comment:
we'd want to key on `events`, not `this` right?

Reply:
Updated to use events